common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -50,7 +50,9 @@
 
                 % if is_radio_input(choice_id):
                   % if status in ('correct', 'partially-correct', 'incorrect') and not show_correctness == 'never':
-                    <span class="sr status" id="${id}-${choice_id}-labeltext">${status.display_name}</span>
+                    <%include file="status_span.html" args="status=status,
+                      classname=status.classname if show_correctness != 'never' else 'unanswered',
```

Comment:
If show_correctness is 'never', what is the display_name? I want to make sure that we don't "leak" the correctness value in any way in that case.

Can you link a problem with this configuration on your sandbox?